eng/tools/spec-gen-sdk-runner/package-lock.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
You shouldn't need a lock file directly in this directory.

Reply:
Removed it.